cli/args/mod.rs

```diff
@@ -101,6 +101,12 @@ pub fn npm_registry_url() -> &'static Url {
   &NPM_REGISTRY_DEFAULT_URL
 }
 
+static JSR_SLOPPY_IMPORTS_AND_NODE_BUILTINS: Lazy<bool> = Lazy::new(|| {
+  std::env::var("JSR_SLOPPY_IMPORTS_AND_NODE_BUILTINS")
+    .ok()
+    .is_some()
+});
```

Comment:
There already exists `DENO_UNSTABLE_SLOPPY_IMPORTS` and `DENO_UNSTABLE_BARE_NODE_BUILTINS`